pgtoolkit/ctl.py

```diff
@@ -246,3 +256,16 @@ def status(self, datadir: Union[Path, str]) -> Status:
         """
         cmd = [str(self.pg_ctl), "status"] + ["-D", str(datadir)]
         return Status(self.run_command(cmd).returncode)
+
+    def get_version(self) -> Tuple[str, int, str]:
```

Comment:
I think this could be computed at instantiation (in `__init__()`) and stored as an attribute.